### DISCUSSION TITLE
r/city-china-jiangsu-nantong-rugao

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#88] [Announcements]
# r/city-china-jiangsu-nantong-rugao

带你探索那些不知名的烟火小城

https://blog.ops-coffee.cn/r/city-china-jiangsu-nantong-rugao

**Comment by thetbw** [comment]
如皋去过一次，也不是旅游，就自驾找朋友去商场的一个漫展，感觉当地路挺宽敞，车也不多

**Comment by JvvYou** [comment]
推荐你试试如皋的萝卜皮😋

**Comment by LessSugarClub** [comment]
之前周末也去如皋逛过李昌钰博物馆、水绘园和定慧禅寺。刑侦博物馆怎么说呢，重在普及李昌钰的人生，主要功绩以及一些刑侦意识、器械和知识。但馆里陈列的一些案件确实是没有结论的。
